boards/posix/native_sim/doc/index.rst

```diff
@@ -436,6 +436,22 @@ The following peripherals are currently provided with this board:
   The flash content can be accessed from the host system, as explained in the
   `Host based flash access`_ section.
 
+**Input events**
+  A driver is provided to read input events from a Linux evdev input device and
+  inject them back into the Zephyr input subsystem.
+
+  The driver is automatically enabled when :kconfig:option:`CONFIG_INPUT` is
+  enabled and the devicetree contains a node such as:
+
+.. code-block:: dts
+
+   evdev {
+     compatible = "input-evdev";
+   };
+
+  The application then has to be run with a flag to specify which evdev device
```

Comment:
Nit: Maybe "flag" => "command line option"? (just to be more specific)

Reply:
ok fixed